src/main/resources/org/jenkinsci/plugins/ParameterizedRemoteTrigger/auth2/CredentialsAuth/config.jelly

```diff
@@ -0,0 +1,7 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project" xmlns:c="/lib/credentials">
```

Comment:
It won't pass injected tests, no Escape-by-default definition. Applies to all files below